DayrSuryan/DSEthiop7.xml

```diff
@@ -255,88 +280,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
                         <seg type="script"/>
+                        <seg type="ink">Black, red.</seg>
                      </handNote>
                   </handDesc>
 
 
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#ivv"/>
-                           <desc type="OwnershipNote"/>
-                           <q xml:lang="gez">ዝመጽሐፍ፡ ዘኀለቃ፡ <persName ref="PRS12557Barha" cert="medium">በርሄ</persName>
+                           <locus target="#iv"/>
+                           <desc type="OwnershipNote">Note stating the manuscript belongs to <persName ref="PRS12557Barha"/>,
+                           whose name is not written clearly. The ink of the second and third characters is smeared.</desc>
+                           <q xml:lang="gez">ዝመጽሐፍ፡ ዘ<persName ref="PRS12557Barha" role="owner"><roleName type="title">ኀለቃ፡</roleName> በሬሄ</persName>
                            </q>
-                           <note>The name of <persName ref="PRS12557Barha">ḫalaqā Barha</persName>is
-                            written not quite clearly,
-                             the ink of the second and
-                             third letters is smeared. </note>
                         </item>
 
                         <item xml:id="a2">
                            <locus target="#ivv"/>
-                           <desc type="OwnershipNote"/>
+                           <desc type="OwnershipNote">Arabic note of the name of <persName ref="PRS11959Pawlos">Būlus al-Ḥabašī</persName>.</desc>
```

Comment:
Stricktly speaking, it is not an ownership note; just note mentioning the name (of the owner), "Pawlos, Abyssinian monk"; as such, it does not state the ownership.